orc8r/cloud/go/services/certifier/protos/certifier.proto

```diff
@@ -83,6 +156,36 @@ service Certifier {
   rpc GetAll(Void) returns (CertificateInfoMap) {}
 
   // cleanup expired certificates
-  //
   rpc CollectGarbage (Void) returns (Void) {}
+
+  // Returns a policy decision given a token, the request method's action
+  // (read/write), and the request resource
+  rpc GetPolicyDecision (GetPolicyDecisionRequest) returns (PolicyDecision) {}
+
+  // Create a new user with their username and password
+  rpc CreateUser (User) returns (Void) {}
```

Comment:
Nit: always use `rpc <Func> (<Func>Request) returns (<Func>Response) {}`

Even if the Req/Res is empty, is good practice for forward-compatibility